usr/share/lib/img_proof/tests/conftest.py

```diff
@@ -163,6 +164,14 @@ def f(provider):
     return f
 
 
+@pytest.fixture()
+def get_baseproduct():
```

Comment:
This would be better combined with [is_sles_sap](https://github.com/SUSE-Enceladus/img-proof/blob/master/usr/share/lib/img_proof/tests/SLES/conftest.py#L351) fixture for now. We only distinguish SAP and not SAP so I don't think we need a function to get product specifically.